cv32e40p/tests/programs/custom/coremark/test.yaml

```diff
@@ -1,4 +1,21 @@
+# Copyright 2020 OpenHW Group
```

Comment:
# Copyright 2020, 2023 OpenHW Group

Note: I would see no copyright statements on such simple yaml files as ok, as the content doesn't have a sufficient creation level. But if you put them, put them right

Reply:
i dont follow. the header in the yaml file state 2020 only. as you modified it in 2023, i would expect this to be reflected